src/modules/dashboard/sagas/colony.ts

```diff
@@ -62,6 +70,14 @@ function* colonyAvatarRemove({
       meta,
       payload: undefined,
     });
+
+    yield apolloClient.query<ColonyQuery, ColonyQueryVariables>({
+      query: ColonyDocument,
+      variables: {
+        address: colonyAddress,
+      },
+      fetchPolicy: 'network-only',
+    });
```

Comment:
This should be before  you send the "success" action

Reply:
Moved it to before & it's still not working. Maybe something with my environment... Is it working on your machine?

Reply:
I just realized this doesn't matter since I'm ripping it out anyway in #2353  :)

So just don't worry about it.